clients/orderbook/src/worker.rs

```diff
@@ -440,6 +440,7 @@ where
 					}
 
 					let withdrawals = self.pending_withdrawals.clone();
+					//TODO: Extra logic to handle it
```

Comment:
Which logic should be here before removing `pending_withdrawals`? If it should not be implemented in this PR - please create issue and add inline comment with reference to that issue.